lib/coverband/collectors/view_tracker.rb

```diff
@@ -10,22 +10,20 @@ module Collectors
     # This is a port of Flatfoot, a project I open sourced years ago,
     # but am now rolling into Coverband
     # https://github.com/livingsocial/flatfoot
-    #
-    # TODO: test and ensure slim, haml, and other support
     ###
     class ViewTracker
-      DEFAULT_TARGET = Dir.glob("app/views/**/*.html.erb").reject { |file| file.match(/(_mailer)/) }
       attr_accessor :target, :logged_views, :views_to_record
       attr_reader :logger, :roots, :store, :ignore_patterns
 
       def initialize(options = {})
         raise NotImplementedError, "View Tracker requires Rails 4 or greater" unless self.class.supported_version?
+        raise "Coverband: view tracker initialized before configuration!" if !Coverband.configured? && ENV["COVERBAND_TEST"] == "test"
 
         @project_directory = File.expand_path(Coverband.configuration.root)
         @ignore_patterns = Coverband.configuration.ignore
         @store = options.fetch(:store) { Coverband.configuration.store }
         @logger = options.fetch(:logger) { Coverband.configuration.logger }
-        @target = options.fetch(:target) { DEFAULT_TARGET }
+        @target = options.fetch(:target) { Dir.glob("#{@project_directory}/app/views/**/*.html.{erb,haml,slim}").reject { |file| file.match(/(_mailer)/) }.freeze }
```

Comment:
I know this is not related to this PR but does the reject of `_mailer` mean we don't support email templates?

Reply:
yeah I can look at it in another PR, I believe in Rails back in the day (Rails 3) we didn't receive active support notifications for email templates... I believe that has been fixed so this could be updated.